pkg/plugin/query.go

```diff
@@ -0,0 +1,100 @@
+package plugin
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+	"time"
+)
+
+const (
+	instantQueryPath = "/api/v1/query"
+	rangeQueryPath   = "/api/v1/query_range"
+)
+
+// Query represents backend query object
+type Query struct {
+	RefId         string `json:"refId"`
+	Instant       bool   `json:"instant"`
```

Comment:
For alerting, we have two types of requests on the panel it is Instant and range. It can be instant or range, but from the response, we always get 2 fields. That is why I defined both. In the main panel, we have three types of requests - instant, range, and both. I think that is why they have this two bool in the JSON